templates/advantage/index.html

```diff
@@ -19,19 +19,19 @@ <h1>Ubuntu Advantage for Infrastructure</h1>
 <section class="p-strip" style="overflow:visible;">
   <div class="row">
     <div class="col-10">
-      <h3>You are signed in</h3>
+      <h2>You are signed in</h2>
       <p>{{ openid.fullname }} ({{ openid.email }})</p>
       <a href="/logout?return_to={{ request.host_url }}advantage" class="p-button--neutral u-no-margin--bottom">Sign out</a>
     </div>
   </div>
 
   <div class="u-fixed-width">
-    <hr class="p-separator" style="margin: 2rem 0;">
+    <hr class="p-separator">
   </div>
 
   <div class="row">
     <div class="col-6 u-sv3">
-      <h4>Your personal subscription</h4>
+      <h2>Your personal subscription</h2>
```

Comment:
As shown in the wireframes, this should be “Your free personal subscription”. I’m sorry I didn’t update the copydoc to match until this morning.

Reply:
Done